.github/config.yml

```
@@ -0,0 +1,25 @@
+newPRWelcomeComment: >
+  Yay, your first pull request towards Jenkins core was created successfully!
+  Thank you so much!
+  <br>
+  <br>
+  A contributor will be by to provide feedback soon.
```

The "be by" has a very nice informal tone to it, but it may not be a common enough  usage to be familiar to those who are not native English speakers or are not very familiar with English.  This comment is optional and is purely a style comment.  I approve whether or not this proposal is accepted.
```suggestion
  A contributor will provide feedback soon.
```